lib/msf/ui/console/command_dispatcher/evasion.rb

```diff
@@ -50,22 +50,32 @@ def cmd_rerun(*args)
 
   alias cmd_rexploit cmd_rerun
 
-  def cmd_exploit_tabs(str, words)
+  #
+  # Tab completion for the run command
+  #
+  def cmd_run_tabs(str, words)
     fmt = {
-      '-e' => [ framework.encoders.map { |refname, mod| refname } ],
-      '-f' => [ nil                                               ],
-      '-h' => [ nil                                               ],
-      '-j' => [ nil                                               ],
-      '-J' => [ nil                                               ],
-      '-n' => [ framework.nops.map { |refname, mod| refname }     ],
-      '-o' => [ true                                              ],
-      '-p' => [ framework.payloads.map { |refname, mod| refname } ],
-      '-t' => [ true                                              ],
-      '-z' => [ nil                                               ]
+        '-e' => [ framework.encoders.map { |refname, mod| refname } ],
+        '-f' => [ nil                                               ],
+        '-h' => [ nil                                               ],
+        '-j' => [ nil                                               ],
+        '-J' => [ nil                                               ],
+        '-n' => [ framework.nops.map { |refname, mod| refname }     ],
+        '-o' => [ true                                              ],
+        '-p' => [ framework.payloads.map { |refname, mod| refname } ],
+        '-t' => [ true                                              ],
+        '-z' => [ nil                                               ]
     }
-    tab_complete_generic(fmt, str, words)
+    flags = tab_complete_generic(fmt, str, words)
+    options = tab_complete_option(str, words)
+    return flags + options
```

Comment:
`return` isn't needed here, in ruby the last expression's evaluated result gets returned implicitly

Reply:
Removed the `return` from the other modules as well.